.github/workflows/cluster_deploy.yaml

```diff
@@ -55,6 +55,7 @@ jobs:
           declare -i COUNTER=5
           until [[ $($k get sac forgerock-sac -o jsonpath='{.status.state}') == "Completed" ]];
           do
+            $k describe sac forgerock-sac
```

Comment:
is it expected to display this ? 
(not related to the subject of the PR)

Reply:
I'm just testing, the tests are failing